SourceryRuntime/Sources/Definable.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Describes that the object is defined in a context of some `Type`
+public protocol Defined: class {
```

Comment:
`Definition` extends `Parsable` and adds path to the file where it is defined. `Parsable` contains definition sourcekit data. Now only `Type` implements `Definition` and the rest is just `Parsable` but I think it make sense to have just one notion of "definition". For types `definedInType` is the same as parent, so no need to resolve it even. My point is that we should keep one notion of "definition", I'm fine with using different name for current `Definition`, we can maybe even get rid of it as we use it only for `Type` anyway. But then I would rename `Defined` to `Definition`.